README.md

```diff
@@ -1342,6 +1342,7 @@ _See also [Monitoring](#monitoring)_
 
 - [Ambar](https://ambar.cloud) - Document Search Engine (OCR, Store & Search). ([Demo](https://app.ambar.cloud/), [Source Code](https://github.com/RD17/ambar)) `MIT` `Nodejs/Python`
 - [Gigablast](http://www.gigablast.com/) - open source search engine. ([Source Code](https://github.com/gigablast/open-source-search-engine)) `Apache-2.0` `C++`
+- [Jina](https://github.com/jina-ai/jina/) - Cloud-native neural search framework for any kind of data. ([Demo](https://github.com/jina-ai/jina/blob/master/.github/pages/hello-world.md)) `MIT` `Python`
```

Comment:
Please remove the demo link (see https://github.com/awesome-selfhosted/awesome-selfhosted/issues/2459, demo links are for actual interactive demos)

Reply:
Thanks for the heads up! Done!